examples/bri-3/src/bri/zeroKnowledgeProof/agents/circuit.agent.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { CircuitService } from '../services/circuit/circuit.service';
+import { Witness } from '../models/witness';
+
+@Injectable()
+export class CircuitAgent {
+  constructor(private readonly circuitAgent: CircuitService) {}
+
+  public async createProof(input: any): Promise<Witness> {
+    const witness = await this.circuitAgent.createProof(input);
```

Comment:
small thing but you can just `return this.circuitAgent ...` same below

Reply:
Fixed.